syndeo-template/src/main/java/com/google/cloud/syndeo/SyndeoTemplate.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (C) 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.syndeo;
+
+import static avro.shaded.com.google.common.base.Preconditions.checkArgument;
```

Comment:
If we're avoiding using the `com.google.common` package directly, we generally use the Beam version rather than Avro one.

Reply:
woah great catch. thanks Zach!